cloudfront-dist/deployment/preview.env

```diff
@@ -0,0 +1,32 @@
+AWS_ACCOUNT=175264504000
+AWS_DEFAULT_REGION=eu-west-1
+STACK_PREFIX_NAME=preview
+STACK_VERSION=1.12.0
```

Comment:
What is STACK_VERSION vs APP_VERSION ?

Reply:
STACK_VERSION is the version of the cloudfront stack and APP_version is the value displayed in blip footer. Perhaps we could merge both...